.ci/scripts/update_package_version.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+echo "Now updating package.json for version: ${1}"
+
+# This is a 3-stage process because jq seems to stream data and
+# will create an empty file if the write is not done atomically
+mydir=$(mktemp -d "${TMPDIR:-/tmp/}$(basename $0).XXXXXXXXXXXX")
+jq '.version = '\"${1}\" package.json > $mydir/package.json
```

Comment:
Will this keep the layout of the rest of the `package.json` file intact or will it be completely re-written in a new style?

Reply:
The file is entirely re-written using the [standard output defaults of jq](https://stedolan.github.io/jq/). If you have a particular style you would like to adhere to, we can try to get `jq` to adhere to it. If we can't get that to happen, we can try other approaches.

Reply:
TL;DR: I just did a test with my version of `jq` and it seems like it doesn't rewrite anything else than what we ask it to so all is good 👍👍👍 ☺️

This, of course, isn't a deal-breaker, but there's a defacto standard in the Node.js community to have a certain order of the important fields in `package.json`. From a technical perspective, it doesn't matter, but if a user opens the file, it makes the readability a bit harder if it doesn't keep the current structure (e.g. first field is always `name`, 2nd is `version` etc.).